### ISSUE TITLE
Sheet metal weapons

### ISSUE THREAD

**Issue by Jarewill** [CleverRaven/Cataclysm-DDA#61001]
### Is your feature request related to a problem? Please describe.

I absolutely love the recent addition of sheet metal armor, it fits the post-apocalyptic knight aesthetic perfectly.
However there aren't really any fitting weapons to the armor set.

### Solution you would like.

Some weapons made from cut sheet metal.
For example a sword made from a metal sheet cut down to a sword-like shape with a hacksaw with a rag-- cotton patch wrapped around for the grip.
Or a metal axe head, again cut from a sheet, attached to a pipe/stick with some duct tape or something stronger, could be flimsy.
Basically just a way to make a metal cutting weapon without the need for a forge or welding.

### Describe alternatives you have considered.

Only staying with a pipe mace and no metal cutting weapons.

**Comment by TheShadowFerret** [comment]
@Aerin-of-the-Toast
